packages/http-server-csharp/src/lib.ts

```diff
@@ -0,0 +1,74 @@
+import { createTypeSpecLibrary, JSONSchemaType, paramMessage } from "@typespec/compiler";
+
+export interface CSharpServiceEmitterOptions {
+  /**Skip formatting of output. Default is false (generated c-sharp files are formatted) */
+  "skip-format"?: boolean;
+  /** Choose which service artifacts to emit.  Default is 'all'.*/
+  "output-type"?: "models" | "all";
+}
+
+const EmitterOptionsSchema: JSONSchemaType<CSharpServiceEmitterOptions> = {
+  type: "object",
+  additionalProperties: false,
+  properties: {
+    "skip-format": {
+      type: "boolean",
+      nullable: true,
+      description:
+        "Skips formatting of generated C# Types.  By default, C# files are formatted using 'dotnet format'.",
+    },
+    "output-type": {
+      type: "string",
+      enum: ["models", "all"],
+      nullable: true,
+      default: "all",
+      description:
+        "Chooses which service artifacts to emit. choices include 'models' or 'all' artifacts.",
+    },
+  },
+  required: [],
+};
+
+export const libDef = {
```

Comment:
```suggestion
export const $lib = createTypeSpecLibrary({
```
nit i would combine that with the next line so you can autocomplete for the structure